share/spack/qa/run-build-tests

```diff
@@ -31,4 +31,4 @@ fi
 spack config get compilers
 
 # Run some build smoke tests, potentially with code coverage
-${coverage_run} bin/spack install ${SPEC}
+${coverage_run} bin/spack.py install ${SPEC}
```

Comment:
The issue is not tied to Debian, but to how `ruamel.yaml` is packaged. The trigger of the bug is to have a recent enough version of `ruamel.yaml` installed in the site directory of your interpreter. More info in https://github.com/spack/spack/issues/9206#issuecomment-420673517

We need to have the python script up here because `coverage run` expects a python script as its argument (it can't go through arbitrary layers of other languages). We are ensured that we won't hit the issue because we control the test environment and we don't:
```console
$ pip install ruamel.yaml
```